ndex2/client.py

```diff
@@ -451,9 +542,30 @@ def get_network_aspect_as_cx_stream(self, network_id, aspect_name):
 
     def get_neighborhood_as_cx_stream(self, network_id, search_string, search_depth=1, edge_limit=2500, error_when_limit=True):
         """
+
         Get a CX stream for a subnetwork of the network specified by UUID network_id and a traversal of search_depth
         steps around the nodes found by search_string.
 
+        .. code-block:: python
+
+
+		import json
```

Comment:
I dont think `import json` is needed in this code fragment. Please check and remove if not needed.

Reply:
Could you try this code fragment without the `import json`? I'm guessing it will work

Reply:
it works without JSON
So Jeff